### PR TITLE
#185106868 A buyer should be able to add items to their wishlist

### PR DESCRIPTION
**Pull request by divinecharlotte**
### What does this PR do?

- This PR is for  allowing a buyer to be able to add items to their wishlist
### How should this be manually tested?

**Installation**
**1. Clone the repository by running** 
```
git clone https://github.com/atlp-rwanda/destructors-ec-fe.git
```
**2. Navigate to the project directory by running **
```
cd destructors-ec-fe
```
```
git checkout ft-add-item-to-wishlist
```
**3. Install the necessary dependencies by running **
```
npm install
```
**6. Run react  development server by running**
```
npm run dev
```

# #  Some background context?
go to the heart icon in the nav or go to  this route `http://localhost:5173/product-wishes` to check your wishlist 

### Questions:

- none

### CODE REVIEW

**Review by ba-lambert — CHANGES_REQUESTED** [review]
**No PR description is around**
```
Why you are violating eslint rules cz it will be harder and harder for us to commit when we rebase your changes
```
> Hover should not change the design of a product card at landing page
what expected: 
Screenshot 2023-06-10 132701


> How a user will be navigated to wishlist page?? (sugention: add an icon on nav bar for wish lised products navigation)

> How a user should remove a wished product to wishlist page?? (sugetion:  add an icon to unwish product also should be visible when we hover a product)
what you have
Screenshot 2023-06-10 133000
how you should do it 
Screenshot 2023-06-10 133801
> Single product add to cart is not working you have to work on its functionality and change bg when product is already wished

💪 Happy coding

File: src/components/products/Product.jsx [hunk 1]
```diff
@@ -3,64 +3,211 @@
...
+//           ) : (
+//             <p className="flex justify-center">Waiting for products</p>
+//           )}
+//         </div>
+//       )}
+//     </>
+//   );
+// }
+
+// export default Product;
```
Comment (Abdulkeza) [inline src/components/products/Product.jsx]: Please could you remove this, we don't really need it

**Review by Abdulkeza — CHANGES_REQUESTED** [review]
Try to clean your code Divine

**Review by buka1calvin — APPROVED** [review]

**Review by mjeanbosco19 — CHANGES_REQUESTED** [review]
Good morning @divinecharlotte , Great Job! I one changes I request on your PR is to automatically increment or decrement number of wished products on the Wishes Icon. As you can see the number on the Icon does dynamically updated.

**Review by ba-lambert — APPROVED** [review]

**Review by mjeanbosco19 — APPROVED** [review]

**Review by Abdulkeza — APPROVED** [review]

**Pull request merged**